src/main/resources/application.properties

```diff
@@ -2,4 +2,19 @@ spring.mvc.converters.preferred-json-mapper=jsonb
 
 spring.security.oauth2.client.registration.github.client-id=${gh_client_id}
 spring.security.oauth2.client.registration.github.client-secret=${gh_client_secret}
-spring.security.oauth2.client.registration.github.scope=repo
\ No newline at end of file
+spring.security.oauth2.client.registration.github.scope=repo
+
+spring.security.oauth2.client.registration.gitlab.client-id=${gl_client_id}
+spring.security.oauth2.client.registration.gitlab.client-secret=${gl_client_secret}
```

Comment:
@criske can you make a short PR in the ``self-xdsd/assets`` repository and write these 2 values there? :D

Reply:
@amihaiemil sure :)